pkgs/development/python-modules/ptvsd/default.nix

```
@@ -0,0 +1,22 @@
+{ stdenv, buildPythonPackage, fetchPypi, pytest }:
+
+buildPythonPackage rec {
+  pname = "ptvsd";
+  version = "4.3.2";
+
+  format = "wheel";
+
+  src = fetchPypi {
+    inherit pname version format;
+    sha256 = "459137736068bb02515040b2ed2738169cb30d69a38e0fd5dffcba255f41e68d";
+  };
```

wheel is less preferable to source, there's a zip download
```suggestion
  src = fetchPypi {
    inherit pname version;
    extension = "zip";
    sha256 = "459137736068bb02515040b2ed2738169cb30d69a38e0fd5dffcba255f41e68d";
  };
```

---

please don't forget to change the hash